Dockerfile

```diff
@@ -12,13 +12,14 @@ ADD requirements.txt /env/requirements.txt
 RUN pip3 install -r /env/requirements.txt
 
 # Dump the app in.
-ADD app/ /app/
+ADD . /app
```

Comment:
Dumping everything into the container ships our test suite, shell scripts and every other development artifact with the app image itself. What's prompting you to do this?

Reply:
Good catch, that change shouldn't have made it in! Fixed it now.